.gitignore

```diff
@@ -171,3 +171,4 @@ garak_runs/
 runs/
 logs/
 .DS_Store
+Miniforge3.sh
```

Comment:
This seems to be a very specific file and should not be in the ignore for the repo, consider moving this to your `.gitignore_global`.

Reply:
Resolved